satellite/satellitedb/usagerollups.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/satellite/console"
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+// usagerollups implements console.UsageRollups
+type usagerollups struct {
+	db *dbx.DB
+}
+
+// GetProjectTotal retrieves project usage for a given period
+func (db *usagerollups) GetProjectTotal(ctx context.Context, projectID uuid.UUID, since, before time.Time) (usage *console.ProjectUsage, err error) {
```

Comment:
Use a custom query instead.

Reply:
U mean 1 custom for everything with join and etc?